content/version/2/0/code_of_conduct.de.md

```diff
@@ -0,0 +1,136 @@
++++
+version = "2.0"
+aliases = ["/version/2/0/de"]
++++
+
+# Vereinbarung über Verhaltenskodex für Mitwirkende
+
+## Unsere Verpflichtung
+
+Wir als Mitglieder, Teilnehmer und Verantwortliche unserer Gemeinschaft
+verpflichten uns bei der Teilnahme an dem Projekt und unserer Gemeinschaft zu
+einer belästigungsfreien Erfahrung für jeden, unabhängig von Alter, Körpergröße,
+Behinderung, ethnischer Zuordnung, geschlechtlicher Identität und Ausdruck,
+Erfahrungsstufe, Bildung, sozialem Status, Nationalität, persönlicher
+Erscheinung, Rasse, Religion oder sexuelle Identität und Orientierung.
+
```

Comment:
sexuelle_r_ Identität

Reply:
I agree with @danielaKay. However, I would even suggest replacing "... sexueller Identität und Orientierung" with "Geschlechtsidentität und sexueller Orientierung". The term _Geschlechtsidentität_ expresses that there's more to gender identity than just the sexual aspect.